### PR TITLE
Feat landing page

### PR DESCRIPTION
**Pull request by daniele24134**

### CODE REVIEW

**Review by CosmicZanna — APPROVED** [review]

File: website/pages/404.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  }, []);
+
+  return (
+    <div className='not-found'>
+      <h1>Ooops....</h1>
+      <h2>That page cannot be found</h2>
+      <p>Go back to the </p>
+      <Link href='/'>
+        <a>Homepage</a>
+      </Link>
```
Comment (stefanfeldner) [inline website/pages/404.tsx:22]: Nice, you handled 404 page too! 🚀

**Review by stefanfeldner — APPROVED** [review]
Nice, looks good to me!
Why no SCSS?

File: website/components/forms/FriendsFormComponent.tsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
+import Link from 'next/link';
+import React from 'react';
+import styles from '../../styles/forms/friendsForm.module.css';
+
+export const FriendsFormComponent: React.FC = () => {
+  return (
+    <section className={styles.friendsForm}>
+      <Link href='/trip'>
+        <a>
+          <img className={styles.goBack} src='/assets/arrow.svg' alt='' />
```
[the remark this answers is not in the document]
Comment (daniele24134) [reply website/components/forms/FriendsFormComponent.tsx:10]: it takes directly from the public folder

**Review by OmarZubaidi — APPROVED** [review]
I don't know how to comment on files on mobile. There's one file with commented out code. Could we leave at least one of them for visual's sake? Also for the friends, could we have the avatar div be self closing?

**Pull request merged**